op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -227,11 +227,8 @@ func DefaultHardforkScheduleForTag(tag string) *genesis.UpgradeScheduleDeployCon
 	switch tag {
 	case ContractsV160Tag, ContractsV170Beta1L2Tag:
 		return sched
-	case ContractsV180Tag, ContractsV200Tag:
-		sched.ActivateForkAtGenesis(rollup.Holocene)
 	default:
 		sched.ActivateForkAtGenesis(rollup.Holocene)
-		sched.ActivateForkAtGenesis(rollup.Isthmus)
```

Comment:
Adrian and Zach specifically asked that Isthmus be activated by default for devnet purposes. Has isthmus been delayed such that its not going into devnets anytime soon?

Reply:
I can update this so that tagged releases don't activate isthmus, but dev releases do.

Reply:
This is designed to work with production tags, and I'm worried about activating an in-development hardfork with a production tag. It'll lead to confusion later.

Reply:
I'll ping Adrian and Zach about this.

Reply:
Updated this so that it activates Isthmus in dev.